tools/manifest/manifest.py

```diff
@@ -400,8 +400,18 @@ def load(tests_root, manifest, types=None, meta_filters=None):
     return _load(logger, tests_root, manifest, types, meta_filters)
 
 
+__load_cache = {}
+
+
 def _load(logger, tests_root, manifest, types=None, meta_filters=None):
     # "manifest" is a path or file-like object.
+    manifest_path = manifest if isinstance(manifest, string_types) \
```

Comment:
I share @jgraham's [mistrust of line continuations](https://github.com/web-platform-tests/wpt/pull/14004#discussion_r232222841), but [it doesn't look like we can disallow them with linting](https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes). Without that, I don't know if we can fairly reject the use of that syntax...

Reply:
Yes we can :p Please don't use continuation lines, use additional parens instead.